www/firebase-crash.js

```diff
@@ -0,0 +1,13 @@
+var exec = require('cordova/exec');
+
+exports.logError = function (message, success, error) {
+  exec(success, error, "FirebaseCrashPlugin", "logError", [message]);
+};
+
+exports.sendNonFatalCrash = function (message, success, error) {
```

Comment:
From js standpoint what’s the difference of this method from ‘log’? You create an exception in native code so stack trace will be the same every time. Is it really useful?

Reply:
The method 'log' adds additional logging that will be sent with your crash data in case there is a fatal app crash (https://firebase.google.com/docs/crashlytics/customize-crash-reports?authuser=0#add_custom_logs).

The method 'sendNonFatalCrash' logs a non-fatal error message and sent it immediately to Crashlytics categorized as a non fatal crash (https://firebase.google.com/docs/crashlytics/customize-crash-reports?authuser=0#log_non-fatal_exceptions).

Reply:
Alright, I got it. `sendNonFatalCrash` creates a new entry in exceptions list while `log` only provides extra context for any future fatal exception.

In such case I'd prefer to rename `logError` -> `log` and `sendNonFatalCrash` -> `logError`. What do you think?

Reply:
I agree, it reflects the method names of the Crashlytics iOS implementation as well. I will correct that.